transport/localpeer_test.go

```diff
@@ -40,31 +37,30 @@ func _TestSendRecv(t *testing.T) {
 }
 
 func TestDropOnBlockedClient(t *testing.T) {
-	logger := log.New(os.Stdout, "", 0)
-	_, r := LinkedPeers(logger)
+	_, r := LinkedPeers()
 
 	// Check that r -> c drops when full
 	for i := 0; i < linkedPeersOutQueueSize; i++ {
-		r.Send(&wamp.Publish{})
+		r.TrySend(&wamp.Publish{})
 	}
 	done := make(chan struct{})
 	var err error
 	go func() {
-		err = r.Send(&wamp.Publish{})
+		err = r.TrySend(&wamp.Publish{})
 		close(done)
 	}()
 	select {
 	case <-done:
 	case <-time.After(time.Second):
 		t.Fatal("Send should have dropped and not blocked")
 	}
-	if err == nil || !strings.HasPrefix(err.Error(), "client blocked") {
+	if err == nil || err.Error() != "try again" {
```

Comment:
prefer a typed error instead of a string comparison.

Reply:
I would agree if there was a general need to branch on error in using this API.  However, this is unit test code that just makes sure the correct error is returned.  I considered returning `syscall.EAGAIN` aka`syscall.EWOULDBOCK`.